serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/servicerecord/ServiceRecord.java

```diff
@@ -12,12 +15,14 @@
     private ServiceIdentifier serviceIdentifier;
     protected String endpointUrl;
     private String pemCertificate;
+    private List<String> dpeCapabilities;
```

Comment:
Burde kanskje ha denne i EDUServiceRecord i stedet siden vi bruker arv i SR og dette feltet ikke er relevant for DPI, DPV og FIKS?